packages/theming/atlas/src/theme/web/custom-variables.scss

```diff
@@ -72,8 +72,12 @@ $link-hover-color: darken($link-color, 15%);
 //== Typography
 //## Change your font family, weight, line-height, headings and more (used in components/typography)
 
-// Font Family Import (Used for google font plugin in theme creater)
+// Font Family Import (Used for google font plugin in theme creater https://ux.mendix.com/theme-creator.html)
```

Comment:
This link is dead, is it going to be updated or can we remove it?

Reply:
Removed the link